prime-dotnet-webapi/Models/Certifications/LicenseDetail.cs

```diff
@@ -30,5 +30,7 @@ public class LicenseDetail : BaseAuditable
         public PrescriberIdType? PrescriberIdType { get; set; }
 
         public bool AllowRequestRemoteAccess { get; set; }
+
+        public string NotPrescPrefix { get; set; }
```

Comment:
I think this should be `NonPrescribingPrefix` as in

![image](https://user-images.githubusercontent.com/7586665/183298037-ac3ee57b-f5d2-44d5-a417-241365e70114.png)